packages/localize/src/date/getMonthNames.js

```diff
@@ -0,0 +1,25 @@
+import { normalizeDate } from './normalizeDate.js';
+
+const monthsLocaleCache = {};
+
+export function getMonthNames({ locale }) {
```

Comment:
Did we deliberately choose not to make notation here configurable?

Reply:
nope

Reply:
it was just not needed

if you look at the implementation of both new functions, you'll see that we can add other `style` options later without changing the API, so let's keep it open for now

Reply:
@tlouisse please create an issue to implement support for other notations of month names